tests/boot/boot_from_pxe.pm

```diff
@@ -137,38 +170,44 @@ sub run {
         type_string_slow(" vt.color=0x07 ");
     }
 
+    wait_still_screen(stilltime => 5, timeout => 120, similarity_level => 50);
     send_key 'ret';
+    wait_still_screen(stilltime => 5, timeout => 120, similarity_level => 50);
     save_screenshot;
 
     if (check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) {
-        my $ssh_vnc_wait_time = 300;
-        my $ssh_vnc_tag       = eval { check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc' } . '-server-started';
-        my @tags              = ($ssh_vnc_tag, 'orthos-grub-boot-linux');
-        assert_screen \@tags, $ssh_vnc_wait_time;
-        save_screenshot;
-        sleep 2;
-
-        if (match_has_tag("orthos-grub-boot-linux")) {
+        if ($is_initrd) {
+            assert_screen 'orthos-grub-boot-linux';
             my $image_name = eval { check_var("INSTALL_TO_OTHERS", 1) ? get_var("REPO_0_TO_INSTALL") : get_var("REPO_0") };
             my $args       = "initrd auto/openqa/repo/${image_name}/boot/${arch}/initrd";
             wait_still_screen 5;
             type_string $args;
             send_key 'ret';
-            assert_screen 'orthos-grub-boot-initrd', $ssh_vnc_wait_time;
+            assert_screen 'orthos-grub-boot-initrd', 100;
```

Comment:
You have reduced timeout. I haven't checked the history, but usually it gets bumped for the reason, at least I hope so =)

Reply:
I have not experienced particularly this part as a problem during testing and I have used the OSD workers. But let's see

Reply:
I think 100 sounds a bit too optimistic, but we'll have to see I guess...